Makefile

```diff
@@ -114,15 +117,19 @@ copy: $(COPY_TARGETS)
 .PHONY: ocamlrun
 ocamlrun: $(OCAMLRUN)
 
-$(BOOT)/ocamlc: $(COPY_TARGETS)
+.PHONY: makedepend
+makedepend: $(CONFIG)
```

Comment:
It seems like the rule for `makedepend` is duplicated, with another one above with different dependencies.